uiowa/uiowa
# Feature: Housing filter fixes

Added no results text, sorting by title, and reducing duplicates for furnishings. 

# How to test

- `ddev blt ds --site=housing.uiowa.edu`
- Go to https://housing.uiowa.ddev.site/residence-halls and verify halls are displayed alphabetically. 
- Verify that you can see no results text and that the view header still displays if there are no results. 
- Try the combinations below and verify that there are not duplicate entries listed in the results. 
- Try to create duplicates with any other searches... 

<img width="420" alt="Screen Shot 2022-07-14 at 9 18 37 AM" src="https://user-images.githubusercontent.com/1036433/179009378-8303f01d-5761-44bf-a946-9078c3e4e1d1.png">
<img width="1519" alt="Screen Shot 2022-07-14 at 9 47 08 AM" src="https://user-images.githubusercontent.com/1036433/179010643-253e7491-4cec-42de-85e9-f230777e2a9e.png">
